README.md

```diff
@@ -22,4 +22,32 @@ A Snap plugin is a program that responds to a set of well defined [gRPC](http://
 You will find [example plugins](examples) that cover the basics for writing collector, processor, and publisher plugins in the examples folder.
 
 
-
+## Snap Diagnostics
+Snap plugins using plugin-lib-go can be run independent of Snap to show their current running diagnostics. This diagnostic information includes:
```

Comment:
Yes, we only have diagnostic information for collector plugins at this time. And the diagnostic capabilities will only be available to plugins running plugin-lib-go.